pkgs/tools/networking/network-manager/default.nix

```diff
@@ -63,31 +55,45 @@ in stdenv.mkDerivation rec {
       inherit (stdenv) shell;
     })
 
+    # Meson does not support using different directories during build and
+    # for installation like Autotools did with flags passed to make install.
+    ./fix-install-paths.patch
+
+    # Our gobject-introspection patches make the shared library paths absolute
+    # in the GIR files. When building docs, the library is not yet installed,
+    # though, so we need to replace the absolute path with a local one during build.
+    # We are replacing the variables in postPatch since substituteAll does not support
+    # placeholders.
+    ./fix-docs-build.patch
   ];
 
   buildInputs = [
-    systemd libuuid polkit ppp libndp curl
+    systemd libselinux audit libpsl libuuid polkit ppp libndp curl
```

Comment:
Do we really want `selinux`?

Reply:
I usually try to have the largest broadest set of features that builds and there is some interest in SELinux in Nix. It does not really affect closure size much (Δ < 1M) but if there are some downsides we can remove it, sure.

```
meson + selinux: 403727760
meson:           403141328
autotools:       413526024
```

Reply:
Yeah that makes sense. I don't know how SELinux works, but does enabling it without support in NixOS cause problems? If not this is fine.